inscriptions/ISic0880.xml

```diff
@@ -182,9 +182,9 @@
                     <lb n="3"/>αἰαῖ ἀπολλύμενος δεσπότου ἀμφ’ ὀδύνῃ·
                     <lb n="4"/>αὐτοῦ γὰρ γενόμην καὶ ἐπίτροπος οὔνομα δ’ ἦν μοι
                     <lb n="5"/>δείλαιος Ἀγάθων δακρυόεις ὁ βίος
-                    <lb n="6"/>τῇ πρὸ <num value="1">αʹ</num> νωνῶν Φεβρουαρίων
-                    <lb n="7"/>μετὰ τὴν ὑπατίαν Ὁνορίου τὸ <num value="8">ηʹ</num> καὶ
-                    <lb n="8"/>Θεοδοσίου τὸ <num value="3">γʹ</num> <expan><abbr>Σεββ</abbr><ex>αστῶν</ex></expan>
+                    <lb n="6"/>τῇ πρὸ <num value="1">α</num> νωνῶν Φεβρουαρίων
+                    <lb n="7"/>μετὰ τὴν ὑπατίαν Ὁνορίου τὸ <num value="8">η</num> καὶ
+                    <lb n="8"/>Θεοδοσίου τὸ <num value="3">γ</num> <expan><abbr>Σεβ<surplus>β</surplus></abbr><ex>αστῶν</ex></expan>
```

Comment:
Not surplus! but `<am>`, the 'abbreviation marker' tag - it is correctly there, but we just give ourselves the option to hide it in the edited version of the text:
`<expan><abbr>Σεβ<am>β</am></abbr><ex>αστῶν</ex></expan>`
see examples at: [https://epidoc.stoa.org/gl/latest/trans-abbrevmark.html](https://epidoc.stoa.org/gl/latest/trans-abbrevmark.html)